src/device-registry/utils/insert-measurements.js

```diff
@@ -51,6 +51,7 @@ const insert = async (tenant, transformedMeasurements) => {
       errors.push(e.message);
     }
   }
+
```

Comment:
please ignore such minor changes when making commits, they increase the number of files changed for no serious reason.